core/includes/form.inc

```diff
@@ -1862,6 +1862,8 @@ function form_builder($form_id, &$element, &$form_state) {
     '#required' => FALSE,
     '#attributes' => array(),
     '#title_display' => 'before',
+    // '#description_display' => (isset($element['#type']) && $element['#type'] == 'fieldset') ? 'before' : 'after',
+    '#description_display' => (isset($element['#type']) && $element['#type'] == 'fieldset') ? 'after' : 'before',
```

Comment:
This here is intentional, for testing purposes - it flips all help text throughout the UI from the default `'after'` to before `'before'` for all elements except fieldsets, which is the opposite.

It should be reverted before merge.

Reply:
...with [`c75c51c` (#4434)](https://github.com/backdrop/backdrop/pull/4434/commits/c75c51c9e60101300ecf425024652007118be29a) I've moved this to a settings form within the Seven theme. So now people can test it that way.